### ISSUE TITLE
UI Enhancements

### ISSUE THREAD

**Issue by AjeshRPai** [wordpress-mobile/WordPress-Android#16114]
- [ ] #16115
- [x]  ~Remove `Progress dialog` being shown during adding category as it's a deprecated component~

**Comment by neeldoshii** [comment]
> Remove Progress dialog being shown during adding category as it's a deprecated component

I believe Progress Dialog is no longer used in Add Category. When you add some category --> category name & choose parent category --> click ok. Progress bar is displayed as refreshing the newly added category. Can you verify it? @AjeshRPai @irfano

**Comment by irfano** [comment]
> > Remove Progress dialog being shown during adding category as it's a deprecated component
> 
> I believe Progress Dialog is no longer used in Add Category. When you add some category --> category name & choose parent category --> click ok. Progress bar is displayed as refreshing the newly added category. Can you verify it? @AjeshRPai @irfano

Good catch, @neeldoshii! I also couldn't find any progress dialog being used after a category is added. It seems the screen was updated after this issue. I'm removing it from the issue description.
